themis/spec/asymmetric-keypairs/_index.md

```diff
@@ -11,3 +11,18 @@ Please come back later.
 
 Meanwhile you can read our [key management guide](/themis/crypto-theory/key-management/).
 {{< /hint >}}
+
+Asymmetric keypairs are used by [**Secure Message**](../secure-message/)
+and [**Secure Session**](../secure-session/) cryptosystems.
+
+Asymmetric cryptography imposes certain mathematical requirements on the structure of the keys.
+To preserve the format and content of the keys intact,
+all asymmetric keys are stored in [Soter containers](../common/#soter-container)
+which provide key type identification and prevent accidental corruption.
+
+## Asymmetric key kinds
+
+Currently, Themis supports the following asymmetric keys:
+
+- [EC keys](ecc/) used by Secure Message and Secure Session
+- [RSA keys](rsa/) supported only by Secure Message
```

Comment:
I would explain here, why we use ECC keys by default. The reason is lower key sizes of ECC keys compared to RSA keys having same levels of security: https://www.globalsign.com/en/blog/elliptic-curve-cryptography (we can even add a table from that blogpost). Plus ECC crypto has lower CPU consumption (https://crypto.stackexchange.com/questions/1190/why-is-elliptic-curve-cryptography-not-widely-used-compared-to-rsa). The only advantage of RSA is that it is better established historically

Reply:
> I would explain here, why we use ECC keys by default.

Yeah, makes sense. I'm not sure if we should go into much detail here, but it will be good to note that ECC is preferred to RSA and we support RSA mostly for compliance reasons. Secure Message page could go in more details on what effects the key choice has on the practical performance of the underlying algorithms.

Reply:
I've added a short blurb about EC vs RSA keys, which hopefully provides a bit more context on why Themis supports multiple types of keys and which is the preferred one.